src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractServerTask.groovy

```diff
@@ -653,9 +653,9 @@ abstract class AbstractServerTask extends AbstractLibertyTask {
                 }
                 application.appendNode('appsDirectory', appDir)
                 if (server.looseApplication) {
-                    application.appendNode('applicationFilename', appObj.archiveName + '.xml')
+                    application.appendNode('applicationFilename', appObj.getArchiveFileName().get() + '.xml')
                 } else {
-                    application.appendNode('applicationFilename', appObj.archiveName)
+                    application.appendNode('applicationFilename', appObj.archiveFileName)
```

Comment:
I see quite a few places where you are using `archiveFileName` directly and other places where you use `getArchiveFileName().get()`. Is the difference intentional?